src/Tribe/Shortcodes/User_Event_Confirmation_List.php

```diff
@@ -65,10 +65,22 @@ public function generate( $params ) {
 	 * @param $params
 	 */
 	protected function set_params( $params ) {
+		/**
+		 * Allow filtering of the default limit for the [tribe-user-event-confirmations] shortcode.
+     *
+		 * @since TBD
+		 *
+		 * @param int $default_limit The default limit to use.
+		 */
+		$default_limit = apply_filters( 'tribe_tickets_shortcodes_attendee_list_limit', 100 );
```

Comment:
```suggestion
		/**
		 * Allow filtering of the default limit for the [tribe-user-event-confirmations] shortcode.
		 *
		 * @since TBD
		 *
		 * @param int $default_limit The default limit to use.
		 */
		$default_limit = apply_filters( 'tribe_tickets_shortcodes_attendee_list_limit', 100 );
```